netplotbrain/plot.py

```diff
@@ -126,13 +126,27 @@ def plot(nodes=None, fig: Optional[plt.Figure] = None, ax=None, view: str = 'L',
     # Figure setup
     # Get number of non-legend rows
     nrows, view, frames = _nrows_in_fig(view, frames)
+    
+    #if subtitles is not specified, give each subplot its view name
+    if subtitles == []:
+        for i in range(0, nrows * frames):
+            subtitles.append('auto')
+    
+    if len(subtitles) != nrows * frames:
+        raise ValueError('Lenght subtitles must be the same as number of subplots')
```

Comment:
"Lenght" is incorrectly spelt. But perhaps use the word shape instead of length (because shape can be multiple dimensions)